### PR TITLE
Added fetch for singular pictogram

### PR DESCRIPTION
**Pull request by Simo205j**
Added fetch functionality for a single pictogram based on an ID. Styling has also been slightly altered to fit with the pictogram such that it is more visible on the Ipad. DISCLAIMER: This is only for demonstration purposes for the thursday meeting, it is by no means a final product and is only there to showcase what our initial design sketches would look like with a pictogram. 

## Description

Please include a summary of the changes and the related issue/ticket from our GitHub Projects.

## Type of PR

- Bugfix
- Feature
- Code style update
- Refactor
- Documentation update
- Other (please describe):

## Related Issues/Tickets

Please link the related issues/tickets here.

## Checklist

- [ ] Code follows the code guidelines

### CODE REVIEW

**Review by DanielRBech — APPROVED** [review]
LGMT

File: components/weekoverview_components/activity_components/ActivityItem.tsx [hunk 1]
```diff
@@ -14,8 +15,9 @@ import Reanimated, {
   SharedValue,
   useAnimatedStyle,
 } from "react-native-reanimated";
+import usePictogram from "../../../hooks/usePictogram";
 
-const CONTAINER_HEIGHT = 80;
+const CONTAINER_HEIGHT = 140;
```
[the remark this answers is not in the document]
Comment (Simo205j) [reply components/weekoverview_components/activity_components/ActivityItem.tsx:20]: Yes, den er aaaaaalt for lille hvis vi ikke ændre højden

File: components/weekoverview_components/activity_components/ActivityItem.tsx [hunk 2]
```diff
@@ -173,8 +189,8 @@ const styles = StyleSheet.create({
     flex: 0.6,
   },
   iconContainer: {
-    width: 65,
-    height: 65,
+    width: 120,
+    height: 120,
```
[the remark this answers is not in the document]
Comment (Simo205j) [reply components/weekoverview_components/activity_components/ActivityItem.tsx:193]: det er selve cirklen som piktogrammet er inden i, og yes det er now for ellers kan man ikke se hvad der er på piktorammet

File: hooks/usePictogram.ts
Comment (MertzAndreas) [inline hooks/usePictogram.ts:1]: Tænkte sådan her 

`import {useQuery} from "@tanstack/react-query";
import {fetchPictograms} from "../apis/pictogramAPI";

export default function usePictogram(id) {
    const queryKey = ['pictograms', id];

    const useFetchPictograms = useQuery({
        queryFn: async () => fetchPictograms(id),
        queryKey: queryKey,
    });

    return {
        useFetchPictograms,
    };
}`

**Review by MertzAndreas — APPROVED** [review]

**Pull request merged**